package.json

```diff
@@ -26,7 +27,12 @@
     },
     "homepage": "https://github.com/tonlabs/TON-Surf-Localization#readme",
     "dependencies": {
-        "@tonlabs/uikit.localization": "^2.3.0",
+        "@react-native-clipboard/clipboard": "1.8.4",
+        "@tonlabs/uikit.localization": "2.3.2",
+        "react-native-localization": "2.1.7",
```

Comment:
I recently reduced bundle for Surf, and figured out that for UIKit it's better to place native deps to peerDeps to avoid duplicates in bundle, I guess better to do it here too

Reply:
Actually maybe all deps, that can be duplicated (aka exist as a dependency in Surf)

Reply:
Yep, but this needs for local checking.

Reply:
Agree with @savelichalex. @BigRoma, consider moving these three dependencies into `peerDependencies` as well as `devDependencies`. Also `typescript` should also be moved to `devDependencies` indeed.